test/unit/kernel/src/KernelWithOmpSchedule.cpp

```diff
@@ -28,43 +25,47 @@ struct KernelWithOmpScheduleBase
     {
-        // By default no run-time check is performed
+        // No run-time check is performed
         alpaka::ignore_unused(acc);
         ALPAKA_CHECK(*success, true);
     }
+};
 
-    // Only check when the schedule feature is active
-#if defined _OPENMP && _OPENMP >= 200805 && defined ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED
-    ALPAKA_NO_HOST_ACC_WARNING
-    template<typename TDim, typename TIdx>
-    ALPAKA_FN_ACC auto operator()(alpaka::AccCpuOmp2Blocks<TDim, TIdx> const& acc, bool* success) const -> void
-    {
-        alpaka::ignore_unused(acc);
-        omp_sched_t kind;
-        int actualChunkSize = 0;
-        omp_get_schedule(&kind, &actualChunkSize);
-        auto const actualKind = static_cast<std::uint32_t>(kind);
-        bool result = (expectedSchedule.kind == actualKind) && (expectedSchedule.chunkSize == actualChunkSize);
-        ALPAKA_CHECK(*success, result);
-    }
-#endif
+// Kernel that sets the schedule kind via constexpr ompScheduleKind.
+// Checks that this variable is only declared and not defined, It also tests that alpaka never odr-uses it.
+struct KernelWithConstexprMemberOmpScheduleKind : KernelWithOmpScheduleBase
+{
+    static constexpr auto ompScheduleKind = alpaka::omp::Schedule::Runtime;
 };
 
-// Kernel that sets the schedule via constexpr ompSchedule.
-// Checks that this variable is only declared and not defined, It also tests that
-// alpaka never odr-uses it.
-struct KernelWithConstexprMemberOmpSchedule : KernelWithOmpScheduleBase
+// Kernel that sets the schedule kind via non-constexpr ompScheduleKind.
+struct KernelWithMemberOmpScheduleKind : KernelWithOmpScheduleBase
 {
-    static constexpr auto ompSchedule = expectedSchedule;
+    static const alpaka::omp::Schedule::Kind ompScheduleKind;
+};
+// In this case, the member has to be defined externally
+const alpaka::omp::Schedule::Kind KernelWithMemberOmpScheduleKind::ompScheduleKind = alpaka::omp::Schedule::NoSchedule;
+
+// Kernel that sets the schedule chunk size via constexpr ompScheduleChunkSize.
+// Checks that this variable is only declared and not defined, It also tests that alpaka never odr-uses it.
+struct KernelWithConstexprStaticMemberOmpScheduleChunkSize : KernelWithOmpScheduleBase
+{
+    static constexpr int ompScheduleChiunkSize = 5;
 };
 
-// Kernel that sets the schedule via non-constexpr ompSchedule.
-struct KernelWithMemberOmpSchedule : KernelWithOmpScheduleBase
+// Kernel that sets the schedule chunk size via non-constexpr ompScheduleChunkSize.
+struct KernelWithStaticMemberOmpScheduleChunkSize : KernelWithOmpScheduleBase
 {
-    static const alpaka::omp::Schedule ompSchedule;
+    static const int ompScheduleChiunkSize;
 };
 // In this case, the member has to be defined externally
-const alpaka::omp::Schedule KernelWithMemberOmpSchedule::ompSchedule = expectedSchedule;
+const int KernelWithStaticMemberOmpScheduleChunkSize::ompScheduleChiunkSize = 2;
```

Comment:
type `ompScheduleChiunkSize` should be `ompScheduleChunkSize`

This typo is also in other parts of this diff.